newer versions seem to report VMM as the HW_PRODUCT
when running as VM in VMM

Index: lib/src/facts/resolvers/virtualization_resolver.cc
--- lib/src/facts/resolvers/virtualization_resolver.cc.orig
+++ lib/src/facts/resolvers/virtualization_resolver.cc
@@ -70,7 +70,7 @@ namespace facter { namespace facts { namespace resolve
             make_tuple("oVirt Node",        string(vm::ovirt)),
             make_tuple("HVM domU",          string(vm::xen_hardware)),
             make_tuple("Bochs",             string(vm::bochs)),
-            make_tuple("OpenBSD",           string(vm::vmm)),
+            make_tuple("VMM",               string(vm::vmm)),
             make_tuple("BHYVE",             string(vm::bhyve)),
         };
 
